google/cloud/aiplatform/models.py

```diff
@@ -1505,29 +1511,67 @@ def predict(
                 [PredictSchemata's][google.cloud.aiplatform.v1beta1.Model.predict_schemata]
                 ``parameters_schema_uri``.
             timeout (float): Optional. The timeout for this request in seconds.
+            use_raw_predict (bool):
+                Optional. If set to True, the underlying prediction call will be made
+                against Endpoint.raw_predict(). Currently, model version information will
+                not be available in the prediciton response using raw_predict.
 
         Returns:
             prediction (aiplatform.Prediction):
                 Prediction with returned predictions and Model ID.
         """
         self.wait()
+        if use_raw_predict:
+            raw_predict_response = self.raw_predict(
+                body=json.dumps({"instances": instances, "parameters": parameters}),
+                headers={"Content-Type": "application/json"},
+            )
+            response_text = json.loads(raw_predict_response.text)
+            return Prediction(
+                predictions=response_text["predictions"],
+                deployed_model_id=raw_predict_response.headers[
+                    "X-Vertex-AI-Deployed-Model-Id"
```

Comment:
Interesting, if you can clarify @rosiezou, I'm guessing `X-Vertex-AI-Deployed-Model-Id` is populated when the response is returned?